.github/workflows/pr_automation.yml

```diff
@@ -0,0 +1,61 @@
+name: PR automation
+on:
+  pull_request:
+    types: [labeled, unlabeled]
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    steps: 
+      - name: retrieve label
+        run: |
+          jq '.' $GITHUB_EVENT_PATH
+          LABEL_NAME1=$(jq --raw-output ".label.name" $GITHUB_EVENT_PATH)
+          echo "Label in action is $LABEL_NAME1"
+          echo ::set-env name=LABEL_NAME::$LABEL_NAME1
+      #inspect the PR to get the project name
+      - name: retrieve project name
+        uses: octokit/graphql-action@v2.x
+        id: get_project_name
+        with:
+          query: |
+            query  get_project_name($owner:String!, $repo:String!, $number:Int!) {
+              repository(owner:$owner,name:$repo) {
+                  pullRequest(number: $number) {
+                    projectCards(first: 1) {
+                      nodes {
+                        project {
+                          name
+                        }
+                      }
+                    }
+                  }
+                }
+              }
```

Comment:
Is this formatting necessary? It kind've looks ugly. Can we put some things on the same line? Like `nodes { project { name } }`. We don't want one really long line, but this looks a bit excessive. Just a style concern, non-blocking.